game_app/games/tic_tac_toe/tic_tac_toe_component.py

```diff
@@ -222,9 +219,8 @@ def handle_event(self, event):
         elif event.type == UserEventTypes.GAME_OVER.value:
             self.winnings = event.new_winnings
         elif event.type == MOUSEBUTTONUP:
-            buttons = [self._scene.restart_button, self._scene.main_menu_button]
-            if self.turn != TurnState.NOT_YOUR_TURN:
-                buttons += sum(self._scene.buttons, [])
+            buttons = [self._scene.restart_button, self._scene.main_menu_button] + \
+                      sum(self._scene.tic_tac_toe_buttons, [])
```

Comment:
Why concatenation with empty list?

Reply:
to nie konkatenacja, tylko funkcyjne sum, to robi tak ze bierze liste np [1, 2, 3] i drugi parametr np 0 i potem do drugiego parametru dodaje po kolei elementy listy i zwroci dla tego przydkladu 6. W moim przypadku biore liste list i zmieniam ja na jednowymiarowa liste, wiec zaczynajac od pustej listy dodaje do niej wszystkie listy wchodzace w sklad tej dwuwymiarowej